P202.Training.WCF/RequestsAndResponses/CreateUserRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Agatha.Common;
+using P202.Training.Domain.Models;
+
+namespace P202.Training.WCF.RequestsAndResponses
+{
+    public class CreateUserRequest : Request
+    {
+        public Users Value { get; set; }
```

Comment:
Would you mind renaming the `Value` property to sth more meaningful? Perhaps `User` or `NewUser`